src/components/BottomBanner/BottomBanner.component.tsx

```diff
@@ -0,0 +1,55 @@
+import { Link, Typography } from '@mui/material';
+import Box from '@mui/material/Box';
+import React, { useState } from 'react';
+import closeIcon from '../../assets/icons/cross.svg';
+
+export const BottomBanner = () => {
+  const [bannerShow, setBannerShow] = useState(true);
+
+  return bannerShow ? (
```

Comment:
What do you think about this?
```suggestion
  if (!bannerShow) {
    return null
  }
  
  return <Box ...
```